app/src/pages/inside/settingsPage/settingTabs/notificationsTab/forms/formControls/formControls.scss

```diff
@@ -0,0 +1,33 @@
+.control-panel {
+  position: relative;
```

Comment:
Please follow the sequence of css rules definition:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Please check all your styles.

Reply:
Fixed